http/health.go

```diff
@@ -3,11 +3,13 @@ package http
 import (
 	"fmt"
 	"net/http"
+
+	platform "github.com/influxdata/influxdb/v2"
 )
 
 // HealthHandler returns the status of the process.
 func HealthHandler(w http.ResponseWriter, r *http.Request) {
-	msg := `{"name":"influxdb", "message":"ready for queries and writes", "status":"pass", "checks":[]}`
+	msg := fmt.Sprintf(`{"name":"influxdb", "message":"ready for queries and writes", "status":"pass", "checks":[], "version": %q}`, platform.GetBuildInfo().Version)
```

Comment:
The more i think about it, the more valuable i feel like `commit` would be in here. (`platform.GetBuildInfo().Commit`) :man_shrugging:

Reply:
A new "commit" field, I can add it add well